### PR TITLE
Taeho-fixed cow bought/sold bug

### PR DESCRIPTION
**Pull request by taeho1031**
In this PR, I modified the frontend and backend so that error message occurs when user tries to buy a cow without enough money or user tries to sell a cow without any cow.

### CODE REVIEW

**Review by jcucsb — COMMENTED** [review]
I think there needs to be one small change implemented. Deployed on Heroku and tested and it seems that when I buy a cow I still get a toast that says "Cow Sold". I think that in the issue it says a toast should only appear when you don't have any cows left to sell. Other than that it works fine when you don't have any cows left to sell. Currently passing CI/CD checks.

**Comment by pconrad** [comment]
frontend run coverage failing with this (and only this)

image

**Comment by pconrad** [comment]
34-frontend-mutation-testing failing with this and only this, which amounts to the same thing:

image

Since that's unrelated to your PR, I'll allow it for now.. but we need to get this fixed.

I'm hopeful that PR #31 from the -4 team might address this... we'll see.

File: frontend/src/main/pages/PlayPage.js [hunk 1]
```diff
@@ -60,10 +60,9 @@ export default function PlayPage() {
       }
     );
   // Stryker enable all 
-
-
+  
   const onSuccessBuy = () => {
-    toast(`Cow bought!`);
+    //toast(`Cow bought!`);
```
Comment (pconrad) [inline frontend/src/main/pages/PlayPage.js:65]: Can we remove this commented out line of code?

File: frontend/src/main/pages/PlayPage.js [hunk 2]
```diff
@@ -90,9 +89,8 @@ export default function PlayPage() {
     mutationbuy.mutate(userCommons)
   };
 
-
   const onSuccessSell = () => {
-    toast(`Cow sold!`);
+    //toast(`Cow sold!`);
```
Comment (pconrad) [inline frontend/src/main/pages/PlayPage.js:93]: Can we remove this commented out line of code?

**Review by pconrad — CHANGES_REQUESTED** [review]

File: frontend/src/main/pages/PlayPage.js (hunk 2 above)
Comment (taeho1031) [reply frontend/src/main/pages/PlayPage.js]: I just removed both "cow bought" and "cow sold" comments!

**Review by jcucsb — DISMISSED** [review]
Works as expected and passes all CI/CD checks. LGTM!

**Review by jcucsb — APPROVED** [review]
No commented out code left and passes all CI/CD checks LGTM!

**Review by jcucsb — APPROVED** [review]
Commented out code removed, passes all CI/CD tests, LGTM

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**